cypress/support/commands.js

```diff
@@ -23,3 +23,32 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.round(Math.random() * 1000);
+  const username = `user${randomNumber}`;
+  const lastname = `user${randomNumber}${randomNumber}`;
+  const email = `${username}@mail.com`;
+  const phone = Math.round(Math.random() * 10000000000);
+  const address = faker.location.streetAddress();
+  const gendersdate = ['Male', 'Female', 'Other'];
+  const hobbiesdate = ['Sports', 'Reading', 'Music'];
+  const subjectdate = ['Math', 'Science', 'History'];
+  const gender = gendersdate[Math.floor(Math.random() * gendersdate.length)];
+  const hobbies = hobbiesdate[Math.floor(Math.random() * hobbiesdate.length)];
+  const subject = subjectdate[Math.floor(Math.random() * subjectdate.length)];
+  // eslint-disable-next-line max-len
+  return {
+    email,
+    username,
+    phone,
+    lastname,
+    address,
+    gender,
+    hobbies,
+    subject
+  };
+}
+
+module.exports = { generateUser };
```

Comment:
I see that you're exporting `generateUser` function using `module.exports`. This is correct for a CommonJS module, but it contradicts with the ES6 `import` syntax you've used on line 26. Make sure you're consistently using either ES6 or CommonJS modules.